tools/src/test/python/dlpx/virtualization/_internal/conftest.py

```diff
@@ -450,6 +453,7 @@ def plugin_manifest(upgrade_operation):
         'hasVirtualPostSnapshot': True,
         'hasVirtualMountSpecification': True,
         'hasVirtualStatus': False,
+        'hasVirtualSourceSize': False,
```

Comment:
Can you please update `tools/src/test/python/dlpx/virtualization/_internal/fake_plugin/direct/successful.py and add this new decorator and plugin implementation on the same lines as other methods here so that pytest under tools covers this?  test_plugin_importer.py uses these fake plugins for unit testing/integ of sorts.

Reply:
Sure, will be taking care in separate JIRA.